hydroflow/examples/kvs_replicated/README.md

```diff
@@ -0,0 +1,39 @@
+A less simple version of the `kvs` example, with multi-node replication. Each server can connect to
+an existing peer server, and updates will be gossiped between them.
+
+Current semantics are:
+ - PUTs are appended: we remember them all forever
+ - GETs are only remembered for the current tick, which may not be monotone depending on how they
+   are consumed.
+ - GETs for empty keys get no acknowledgement.
+
+ Clients accept commands on stdin. Command syntax is as follows:
+ - `PUT <key>, <value>`
+ - `GET <key>`
+ Commands are case-insensitive. All keys and values are treated as `String`s.
+
+## Running the example
+
+We will need four (or more) terminals:
+
+First, start a server:
+```
+cargo run -p hydroflow --example kvs_replicated -- --role "server" --addr localhost:12346
```

Comment:
server is quoted but client is not, it should be consistent between the two documented example lines I think.